app/controllers/wishlists_controller.rb

```diff
@@ -0,0 +1,22 @@
+class WishlistsController < ApplicationController
+  def new
+    @wishlist = Wishlist.new
+  end
+
+  def create
+    @wishlist = Wishlist.new(wishlist_params)
+    @wishlist.user_id = current_user.id
+    if @wishlist.save
+      flash[:notice] = "Success"
+      redirect_to dashboard_path
```

Comment:
Also, based off the thoughtbot best practices in our style guide, consider using `dashboard_url` here instead of `dashboard_path`. See https://github.com/thoughtbot/guides/tree/master/best-practices#rails

> Use _url suffixes for named routes in mailer views and redirects. Use _path suffixes for named routes everywhere else.

https://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html#sec14.30